File: dmytryshyn/Lab3/Lab3/Lab3/Classes/Ships/Implementations/HeavyShip.cs
using MySecondPattern.Classes.Ships;
using MySecondPattern.Classes.Ships.Characteristics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab3.Classes.Ships.Implementations
{
    public class HeavyShip : Ship
    {
        public HeavyShip(int id, double fuel, ShipLimitsCharacteristics limitsCharacteristics) : base(id, fuel, limitsCharacteristics)
        {
        }
        public override string ToString()
        {
            return $"The ship {Id} is a heavy ship.";
        }
    }
}

File: mukha/lab_3/PortManagementSystem/ships/heavy_ship.py
from ships.ship import Ship

class HeavyShip(Ship):
    def calculate_fuel(self):
        # Логіка розрахунку пального для важкого корабля
        self.fuel = self.capacity * 0.3  # Приклад логіки розрахунку пального

    def load_container(self, container):
        # Логіка завантаження контейнера на корабель
        pass
